### ISSUE TITLE
Tag Filter Source Nullpointer

### ISSUE THREAD

**Issue by DigiLiviu** [adobe/asset-share-commons#1022]
Hi,

We recently upgraded to version 3.4.0 and then started getting a nullpointer exception on our search page. I noticed that the nullpointer was pointing to
```
com.adobe.aem.commons.assetshare.components.predicates.impl.PropertyPredicateImpl.getItems(PropertyPredicateImpl.java:138)
```
I debugged the code and saw that it was happening because the `source` object was null at the following line
```
if (source.equals("json"))
```
In looking at the resource object of the predicate, I noticed it was actually on `Tag Filter` component, not a `Property Filter`, which is what the `PropertyPredicateImpl` should be triggering from. I guess it may be happening because the Property Filter is a super resource type of the Tag Filter. The Tag Filter dialog, however, doesn't have a `source` field (it's actually explicitly hidden), so it will always be null if `PropertyPredicateImpl` is triggered.

Are we doing something wrong that it is getting invoked like that? I removed all the Tag Filter components from the page and it did start working, so it certainly seems like they are the culprit.

Thanks,

Liviu

**Comment by DigiLiviu** [comment]
Just noticed that there seems to be a PR in place to fix this at https://github.com/adobe/asset-share-commons/pull/1007

Any plans to make this part of a release?

**Comment by davidjgonzalez** [comment]
Sorry -i thought this was released. Cutting 3.4.2 with this fix now

**Comment by davidjgonzalez** [comment]
Fixed in 3.4.2 released today

**Comment by DigiLiviu** [comment]
Thanks David!
